torchaudio/csrc/sox/utils.cpp

```diff
@@ -209,22 +213,25 @@ namespace {
 
 std::tuple<sox_encoding_t, unsigned> get_save_encoding_for_wav(
     const std::string format,
-    const caffe2::TypeMeta dtype,
+    caffe2::TypeMeta dtype,
```

Comment:
I'm surprised that `toScalarType` is not marked as `const`. Do you think it makes sense to change the function signature to pass around `c10::ScalarType`, instead of `caffe2::TypeMeta`? (`caffe2` itself is absorbed to PyTorch, so seeing it as function signature feels strange too.)